interactions.tsv

```diff
@@ -1355,50 +1355,50 @@ sourceOccurrenceId	sourceTaxonId	sourceTaxonName	sourceBodyPartId	sourceBodyPart
-		Epithemia turgida						host of			EtSB 										10.1073/pnas.1405222111	(Nakayama et al. 2014)
-		Asterionella formosa						has parasite			Chytrids										10.1038/ismej.2013.73	(Gsell et al. 2013)
-		Rhizosolenia hebetata						host of			Richelia intracellularis										WOS:000325596800006	(Madhu et al. 2013)
-		Pseudo nitzschia 						mutualism			Bacteria										10.1038/ismej.2013.138	(Sison-Mangus et al.2014)
-		Bacillariophyta 						host of			Amphistegina radiata										10.4319/lo.2011.56.5.1587	(Schmidt et al. 2011)
-		Nitzschia 						host of			Heterostegina depressa										10.4319/lo.2011.56.5.1587	(Schmidt et al. 2011)(Lee et al. 2013)(Hallock. 2008)
-		Bacillariophyta 						host of			Calcarina hispada										10.4319/lo.2011.56.5.1587	(Schmidt et al. 2011)
-		Skeletonema 						host of			Peridiniopsis kevei										10.2216/07-36.1	(Takano et al. 2008)
-		Skeletonema 						host of			Peridiniopsis penardii										10.2216/07-36.1	(Takano et al. 2008)
-		Nitzschia 						host of			Dinoflagellates										10.2216/07-36.1	(Takano et al. 2008)
-		Thalassionema 						host of			Nummulitidae										WOS:000243459500005	(Holzmann et al. 2006)
-		Diadesmis 						host of			Coenogonium linkii										10.1111/j.0022-3646.2004.02-205.x	(Lakatos et al. 2004)
-		Eunotia 						host of			Coenogonium linkii										10.1111/j.0022-3646.2004.02-205.x	(Lakatos et al. 2004)
-		Hantzschia 						host of			Coenogonium linkii										10.1111/j.0022-3646.2004.02-205.x	(Lakatos et al. 2004)
-		Luticola 						host of			Coenogonium linkii										10.1111/j.0022-3646.2004.02-205.x	(Lakatos et al. 2004)
-		Melosira 						host of			Coenogonium linkii										10.1111/j.0022-3646.2004.02-205.x	(Lakatos et al. 2004)
-		Nitzschia 						host of			Coenogonium linkii										10.1111/j.0022-3646.2004.02-205.x	(Lakatos et al. 2004)
-		Orthoseira 						host of			Coenogonium linkii										10.1111/j.0022-3646.2004.02-205.x	(Lakatos et al. 2004)
-		Pinnularia 						host of			Coenogonium linkii										10.1111/j.0022-3646.2004.02-205.x	(Lakatos et al. 2004)
-		Stauroneis 						host of			Coenogonium linkii										10.1111/j.0022-3646.2004.02-205.x	(Lakatos et al. 2004)
-		Pennate 						host of			Peridinium balticum										WOS:000166156700015	(Inagaki et al. 2000)
-		Bacillariophyta 						host of			Peridinium foliaceum										WOS:000166156700015	(Inagaki et al. 2000)
+		Epithemia turgida					http://purl.obolibrary.org/obo/RO_0002453	host of			EtSB 										10.1073/pnas.1405222111	(Nakayama et al. 2014)
+		Asterionella formosa					http://purl.obolibrary.org/obo/RO_0002445	has parasite			Chytrids										10.1038/ismej.2013.73	(Gsell et al. 2013)
+		Rhizosolenia hebetata					http://purl.obolibrary.org/obo/RO_0002453	host of			Richelia intracellularis										WOS:000325596800006	(Madhu et al. 2013)
+		Pseudo nitzschia 					http://purl.obolibrary.org/obo/GO_0085030	mutualism			Bacteria										10.1038/ismej.2013.138	(Sison-Mangus et al.2014)
```

Comment:
Rather that using http://purl.obolibrary.org/obo/GO_0085030 , suggest to use http://purl.obolibrary.org/obo/RO_0002442 .

Reply:
Got it. I'll make the changes in my local copy.
Do I make a new pull request afterwards?
Never done this specific workflow before :-)